gadget/install/content_test.go

```diff
@@ -290,8 +290,9 @@ func (s *contentTestSuite) testWriteFilesystemContentDriversTree(c *C, isCore bo
 		IsCore:           isCore,
 	}
 
-	restore = install.MockKernelEnsureKernelDriversTree(func(kSnapRoot, destDir string, kmodsConts []snap.ContainerPlaceInfo, opts *kernel.KernelDriversTreeOptions) (err error) {
-		c.Check(kSnapRoot, Equals, kMntPoint)
+	restore = install.MockKernelEnsureKernelDriversTree(func(kMntPts kernel.MountPoints, compsMntPts []kernel.ModulesCompMountPoints, destDir string, opts *kernel.KernelDriversTreeOptions) (err error) {
+		c.Check(kMntPts, Equals,
+			kernel.MountPoints{Current: kMntPoint, Target: kMntPoint})
```

Comment:
do we need a test where these end up being different? looking at the code that we changed in content.go is not clear they will always be the same?

Reply:
I've added a test for this now